<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Passenger</class>
 <widget class="QWidget" name="Passenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(220, 138, 221);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="passengerLabel">
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;
</string>
     </property>
     <property name="text">
      <string>Passenger i</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="stateLabel">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentFloorLabel">
     <property name="text">
      <string>Current Floor</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
